internal/provider/resource_gitlab_topic_test.go

```diff
@@ -370,46 +412,71 @@ func testAccCheckGitlabTopicSoftDestroy(s *terraform.State) (err error) {
 }
 
 func testAccGitlabTopicRequiredConfig(rInt int) string {
+	var titleConfig string
+	if titleIsSupported, err := isGitLabVersionAtLeast(context.Background(), testGitlabClient, "15.0")(); err == nil && titleIsSupported {
+		titleConfig = fmt.Sprintf(`title = "Foo Req %d"`, rInt)
+	}
```

Comment:
Same comment as this pattern last time, seems like 15+ should maybe be a separate test. This matches our pattern of using in-line configs now anyway, right?

Reply:
It's because it's a required attribute in 15.0 and we'd need to duplicate the entire test which has quite some test steps. I don't think it's worth that in this case.

Regarding using the in-line configs: I agree, it should be, but let's do this in a separate PR and focus here on getting 15.0 support :)

Reply:
Works for me :)